src/main/target/SPRACINGF4EVO/target.c

```diff
@@ -26,39 +26,40 @@
 
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    { TIM9,  IO_TAG(PA3),   TIM_Channel_2, TIM_USE_PPM | TIM_USE_PWM,   0, GPIO_AF_TIM9, NULL, 0, 0 }, // PPM / PWM1 / UART2 RX
-    { TIM9,  IO_TAG(PA2),   TIM_Channel_1, TIM_USE_PWM,                 0, GPIO_AF_TIM9, NULL, 0, 0 }, // PPM / PWM2 / UART2 TX
 
-    { TIM8,  IO_TAG(PC6),  TIM_Channel_1, TIM_USE_MOTOR,                1, GPIO_AF_TIM8,  DMA2_Stream2, DMA_Channel_7, DMA2_ST2_HANDLER }, // ESC 1
-    { TIM8,  IO_TAG(PC7),  TIM_Channel_2, TIM_USE_MOTOR,                1, GPIO_AF_TIM8,  DMA2_Stream3, DMA_Channel_7, DMA2_ST3_HANDLER }, // ESC 2
-    { TIM8,  IO_TAG(PC9),  TIM_Channel_4, TIM_USE_MOTOR,                1, GPIO_AF_TIM8,  DMA2_Stream7, DMA_Channel_7, DMA2_ST7_HANDLER }, // ESC 3
-    { TIM8,  IO_TAG(PC8),  TIM_Channel_3, TIM_USE_MOTOR,                1, GPIO_AF_TIM8,  DMA2_Stream4, DMA_Channel_7, DMA2_ST4_HANDLER }, // ESC 4
+    DEF_TIM(TIM9,  CH2, PA3,  TIM_USE_PPM | TIM_USE_PWM,   0, 0), // PPM / PWM1 / UART2 RX
```

Comment:
Can you insert comment with DEF_TIM argument explanation? last 2 arguments are very confusing